static/css/index.scss

```diff
@@ -35,11 +35,20 @@
     }
 
     &__logo {
-      width: 12em;
-      overflow: auto;
-      display: flex;
-      flex-direction: row;
-      padding: 0.5em 1em 0.5em 0.5em;
+
+      a {
+        text-decoration: none;
+        width: 14em;
+        overflow: auto;
+        display: flex;
+        flex-direction: row;
+        padding: 0.5em 1em 0.5em 0.5em;
+        color: $main;
+
+        &:link + :visited + :hover + :active {
```

Comment:
What is this? What does `:link` do? In CSS when you have a `+`, it means subsequent elements, so this is saying a style for a link followed by a `:visited` url followed by a `:hovered` item followed by a `:active` item, which doesn't seem right.